packages/schematics/angular/migrations/update-14/angular-packages-version-prefix_spec.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { EmptyTree } from '@angular-devkit/schematics';
+import { SchematicTestRunner, UnitTestTree } from '@angular-devkit/schematics/testing';
+
+describe('Migration to update Angular packages version prefix to `^` instead of `~`', () => {
+  const packageJsonPath = '/package.json';
+  const schematicName = 'update-angular-packages-version-prefix';
+  const schematicRunner = new SchematicTestRunner(
+    'migrations',
+    require.resolve('../migration-collection.json'),
+  );
+
+  let tree: UnitTestTree;
+  beforeEach(() => (tree = new UnitTestTree(new EmptyTree())));
```

Comment:
Nit: I recommend using typical braces for assignments like this rather than an expression assignment, which is generally only used in optimized builds.

Reply:
Agreed